docs/algorithms/castor/linear-problem/inter-temporal-constraints/generator-constraints-filler.md

```diff
@@ -0,0 +1,184 @@
+
+> Note that for the first timestamp, the initial state $S_{0}(g,s)$ is used as the initial condition of the chain:
+> - if $S_{0}(g,s)$ is On: $T_{ON \to ON}(g,s,1) + T_{ON \to OFF}(g,s,1) + T_{ON \to RD}(g,s,1) + T_{ON \to RU}(g,s,1) =
+    1$
+> - if $S_{0}(g,s)$ is Off: $T_{OFF \to ON}(g,s,1) + T_{OFF \to OFF}(g,s,1) + T_{OFF \to RD}(g,s,1) + T_{OFF \to RU}(
+    g,s,1) = 1$
+
+### Power constraints
+
+#### Off state
+
+By definition, $OFF(g,s,t) = 1 \Rightarrow P(g,s,t) = 0$ which can be linearized as:
+
+$$P(g,s,t) \leq P_{\max}(g) (1 - OFF(g,s,t))$$
+
+#### On state
+
+By definition, $ON(g,s,t) = 1 \Leftrightarrow P(g,s,t) \geq P_{\min}(g)$ which can be linearized using the two following
+equations:
+
+$$P(g,s,t) \leq P_{min}(g) (1 - ON(g,s,t)) + P_{\max}(g) ON(g,s,t)$$
+$$P_{t} \geq P_{\min} ON_{t}$$
+
+#### Power variation
+
+The power of the generator determines the state of the generator. The two previous sections showed how to identify the
+on and off state. In between these are the ramping states which can be determined based on the previous state and/or
+whether the power is increasing (ramp up) or decreasing (ramp down).
+
+More generally, the power variation and the state transitions are strongly entangled and constrain one another.
+The following table displays the bounds of the variation $P(g,s,t) - P(g,s,t-1)$ based on the state transition:
+
+> - A single value in a cell means that said value is both a lower and an upper bound.
+> - A "$\times$" symbol denotes an [impossible transition](#impossible-transitions).
+
+| From / To | $OFF$                                                                              | $RU$                             | $ON$                                                                                                                                                     | $RD$                                                                                                                                                     |
+|-----------|------------------------------------------------------------------------------------|----------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| $OFF$     | 0                                                                                  | $\Delta t \frac{P_{\min}}{LEAD}$ | $P_{\min} \leq . \leq P_{\min} + (\Delta t - \lambda_{t}^{\nearrow}) \nabla^{+}$                                                                         | $\times$                                                                                                                                                 |
```

Comment:
This nabla is the same as the one defined at lines 40-41? In that case, I would suggest to remove the p between the nabla and the +/- at lines 40-41, because it seems that you never use it (the nabla is only used here, and adding a p would make it less easily readable)